pyspider/database/elasticsearch/resultdb.py

```diff
@@ -62,7 +62,7 @@ def select(self, project, fields=None, offset=0, limit=0):
         else:
             for record in self.es.search(index=self.index, doc_type=self.__type__,
                                          body={'query': {'term': {'project': project}}},
-                                         _source_include=fields or [], from_=offset, size=limit,
```

Comment:
Changed to `source_includes`?
https://elasticsearch-py.readthedocs.io/en/latest/api.html#elasticsearch.Elasticsearch.search

Reply:
Ignore this. My bad.